Controllers/getCommentsById.controller.js

```diff
@@ -0,0 +1,10 @@
+const { selectCommentsById } = require("../Models/selectCommentsById.model");
+
+exports.getCommentsById = (req, res, next) => {
+  const reviewID = req.params.review_id;
+  selectCommentsById(reviewID)
+    .then((result) => {
+      res.status(200).send(result);
```

Comment:
here it's best to send back the result in an object, and on a key that fits what it is, for example  { comments: result }, it may seem pointless, but when doing front end project you'll use these endpoints and it's nice to have the data you get back stored on semantic property keys